cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -489,6 +489,11 @@ func isDeviceDataUpdated(oldData *v1alpha2.DeviceData, newData *v1alpha2.DeviceD
 	return !reflect.DeepEqual(oldData, newData)
 }
 
+// isDeviceDataUpdated checks if DeviceData is updated
```

Comment:
isDevicePropertyVisitorsUpdated checks if PropertyVisitors has been updated

Reply:
Thanks, I modify this.